docs/tutorials/src/SUMMARY.md

```diff
@@ -12,5 +12,7 @@
 - [Rendering](10_rendering.md)
 - [Advanced component strategies](11_advanced_component.md)
 - [FlaggedStorage and modification events](12_tracked.md)
+- [Saveload](13_saveload.md)
+- [Troubleshooting](14_troubleshooting.md)
```

Comment:
This was missing before, is this an intentional addition?

Reply:
It is intentional, I assumed the troubleshooting chapter was added but never linked, up to @torkleyy to decide whether this belongs in a different PR.